Common.mk

```diff
@@ -410,17 +436,20 @@ validate-checksums: $(BINARY_TARGETS)
 	$(BUILDCTL)
 
 # Build helm chart
+ifeq ($(HAS_HELM_CHART),true)
 .PHONY: helm/build
-
+helm/build: ## Build helm chart
+helm/build: $(if $(or $(wildcard $(PROJECT_ROOT)/helm/patches),$(wildcard $(MAKE_ROOT)/helm/patches)),$(HELM_GIT_PATCH_TARGET),$(HELM_GIT_CHECKOUT_TARGET))
```

Comment:
I was suggestion just moving `$(OUTPUT_DIR)/ATTRIBUTION.txt` to this line before the if instead of two pre-req lines.  Its a nit, we just usually put all the pre-reqs on the same line, unless they are variable sets.

Reply:
I'm not terribly attached, but as long as it acts the same, I think it is a bit easier to read on three lines.